### PR TITLE
[RM Ch2] Update list of network functions 

### LINKED ISSUE #2066: [RM Ch2.1] Review network applications used in the analysis
From the vF2F in October 2020:

> I note the presence of OCS in 3G/4G but no reference to the CHF/CCS in 5G.  I understand that many Edge based scenarios are expecting deployment of user plane functions but there are also discussions that control plane functions (which includes CHF - Charging Function) might make sense to bring to the edge.

In addition, the "Gathering and Validating CNF Requirements..." session agreed that this section needs a review to ensure any further analysis takes an up-to-date list of network applications as input.

Reference: https://etherpad.opnfv.org/p/Gathering_and_Validating_CNF_Requirements

### PR DESCRIPTION
**Pull request by Smitholi67**
Fixes #2066 
Added CHF/CCS to list of 5G Core NFs (Control Plane).

### CODE REVIEW

File: doc/ref_model/chapters/chapter02.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ The list of, most likely to be virtualised,  Network Functions below, covering a
     - UDR: Unified Data Repository
     - NRF:  Network Repository Function
     - NEF:  Network Exposure Function
+    - CHF/CCS:  Charging Function / Converged Charging System
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter02.md:78]: please split CHF and CCS into their own separate rows.

Comment (Smitholi67) [reply doc/ref_model/chapters/chapter02.md:78]: I can do that, but CHF is a part of the CCS,  CHF interacts on the SBA, the CCS does not.  To me it makes more sense to keep together, or just drop CCS.

Comment (pgoyal01) [reply doc/ref_model/chapters/chapter02.md]: Agree that they are related -- but we have other related functions listed separately.  

It is your decision to make. 

BTW you will also need to add these to the abbreviations list

**Review by walterkozlowski — APPROVED** [review]

**Review by karinesevilla — COMMENTED** [review]
@Smitholi67 @pgoyal01 
CCS is not a function, but a set of functions. 
If CHF is the only function relevant in our scope, I prefer to keep CCS with CHF and not split them.
May be a wording like: ' CHF: charging Function part of the converged charging system'

**Review by pgoyal01 — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]
@Smitholi67 Thanks for the modifications

**Pull request merged**